include/ncpp/NotCurses.hh

```diff
@@ -140,9 +140,9 @@ namespace ncpp
 			return error_guard (palette256_use (nc, reinterpret_cast<const palette256*>(&p)), -1);
 		}
 
-		int render () const NOEXCEPT_MAYBE
+		bool render () const NOEXCEPT_MAYBE
 		{
-			return error_guard<int> (notcurses_render (nc), -1);
+			return error_guard (notcurses_render (nc), -1);
```

Comment:
perhaps i'm misunderstanding how this works, but is -1 still appropriate here? let me go look at the definition of `error_guard` and figure out exactly how it works (i've got it, no need to dumb it down for me assuming this is what was intended).

Reply:
ok TValue and TRet are independent, so nope, this looks fine